dev/test-studio/structure/groupByOption.ts

```diff
@@ -0,0 +1,30 @@
+import {StructureBuilder} from 'sanity/lib/dts/src/_exports/desk'
```

Comment:
this should probably be `sanity/desk` (this happens because of typescript links, i think)
```suggestion
import {StructureBuilder} from 'sanity/desk'
```

Reply:
Yeah, intellij autofix. I'll do a oass o er imports to ensure they are fixed.